AudibleApi/Locale.cs

```diff
@@ -1,10 +1,9 @@
 ﻿using System;
-using System.Collections.Generic;
 using Dinah.Core;
 
 namespace AudibleApi
 {
-	public class Locale : ValueObject
+	public record Locale
```

Comment:
I thought this made more sense as a record with the build-in equality comparison.

Reply:
lol it's all good. That code was written long before this was an option and is clearly trying to approximate the same thing. I highly prefer yours b/c I prefer native solutions. In fact, my old code might be obsolete.